### ISSUE TITLE
Contributor Guide

### ISSUE THREAD

**Issue by Osalotioman** [GDGoC-University-of-Benin/the-intern-place#8]
Please can a CONTRIBUTING.md file be added to the repo, so I know how I can contribute to the project.

**Comment by d-a-ve** [comment]
Yes, it can. I was just about to create an issue for it. Can you work on one?

**Comment by Osalotioman** [comment]
Yes, I can work on one. I'll start working on it as soon as possible.
